src/data/data_fetcher.py

```diff
@@ -116,23 +118,26 @@ def _handle_two_year_transaction_period(
             else:
                 two_year_transaction_period = "2020"
                 print("Invalid input, defaulting to 2020.")
+        elif (
+            int(two_year_transaction_period) >= 2000
+            and int(two_year_transaction_period) <= 2020
+            ):
+            pass  
         else:
             two_year_transaction_period = "2020"
             print("Invalid input, defaulting to 2020.")
     else:
         two_year_transaction_period = "2020"
         print("Invalid input, defaulting to 2020.")
 
-    return two_year_transaction_period
+    return str(two_year_transaction_period)
 
 
 def _handle_recipient_committee_type(recipient_committee_type: str) -> str:
-    if recipient_committee_type == "H" or recipient_committee_type == "House":
-        recipient_committee_type = "H"
-    elif recipient_committee_type == "S" or recipient_committee_type == "Senate":
-        recipient_committee_type = "S"
-    elif recipient_committee_type == "P" or recipient_committee_type == "Presidential":
-        recipient_committee_type = "P"
+    possible_types = ["H", "HOUSE", "S", "SENATE", "P", "PRESIDENTIAL"]
+    recipient_committee_type = recipient_committee_type.upper()
+    if recipient_committee_type in possible_types:
+        return recipient_committee_type[:1]
```

Comment:
This is elegant as hell, I like it :)

Reply:
Honestly when I thought of it that imposter syndrome disappeared for like an hour lol.

Reply:
Be proud of it, man. I look back at my old code and cringe like hell (years old, months old, weeks old. Doesn't matter.)

Don't see any room for improvement on this one 🔥